github/repos_test.go

```diff
@@ -852,3 +852,19 @@ func TestRepositoriesService_RemoveAdminEnforcement(t *testing.T) {
 		t.Errorf("Repositories.RemoveAdminEnforcement returned error: %v", err)
 	}
 }
+
+func TestPullRequestReviewsEnforcementRequest_MarshalJSON_nilDismissalRestirctions(t *testing.T) {
+	req := PullRequestReviewsEnforcementRequest{}
+
+	json, err := json.Marshal(req)
+
+	if err != nil {
```

Comment:
Remove the blank lines between `foo, err := something()` and `if err != nil {`. They hurt readability.